src/Uno.UWPSyncGenerator/DocGenerator.cs

```diff
@@ -155,13 +155,20 @@ void BuildMemberLists()
 								AppendImplementedMembers("methods", "Method", methods);
 								AppendImplementedMembers("events", "Event", events);
 
-								_sb.AppendHorizontalRule();
+								var notImplementedProperties = GetNotImplementedMembers(properties);
```

Comment:
The idea behind the change done here is to not generate a statement that says `Below are all properties, methods, and events of XYZ that are **not** currently implemented in Uno.` when all members are implemented.